tests/functional/keystoneapi_controller_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	. "github.com/onsi/gomega"    //revive:disable:dot-imports
 
 	//revive:disable-next-line:dot-imports
+	"github.com/openstack-k8s-operators/lib-common/modules/common"
 	. "github.com/openstack-k8s-operators/lib-common/modules/common/test/helpers"
```

Comment:
I think the revive disable comment should be on top of the dot import line. Strange that the linter didn't start to complain.